sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/HttpHelperTests.cs

```diff
@@ -309,5 +342,173 @@ public void NetHostNameAttributeTakesPrecedenceSettingHttpRequestUrl()
             string url = PartBTags.GetRequestUrl();
             Assert.Equal(expectedUrl, url);
         }
+
+        [Fact]
+        public void HttpDependencyUrlIsSetUsingHttpUrl()
+        {
+            var PartBTags = AzMonList.Initialize();
+            AzMonList.Add(ref PartBTags, new KeyValuePair<string, object>(SemanticConventions.AttributeHttpUrl, "https://www.wiki.com"));
```

Comment:
nit:
confirm if there exists unit test testing the case whether the expected tag is not added, resulting in null url.

Reply:
are you referring to HttpUrlIsNullByDefault() test?